src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
+    private ColorSupplier supplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(supplier.getRandomColor(), new Random().nextInt(MAX_BALL_NUMBER));
```

Comment:
Think about which variables should be local in the method and which should be class-level. `new Random()` object will be created on each `getRandomBall()` call. How can you avoid this?